components/form-builder/app/Nagware.tsx

```diff
@@ -0,0 +1,106 @@
+import React from "react";
+import { useTranslation } from "next-i18next";
+import { Attention } from "@components/globals/Attention/Attention";
+import { NagLevel, NagwareResult } from "@lib/nagware";
```

Comment:
Be wary of importing `@lib/*` into tsx components.  It has happened in the past where a server side module was later added as an import into a `@lib/*` module which caused errors as it's dependencides could not run on the browser or vice versa.  Might suggest moving the enum and types for Nagware to a `@types` to be safe.

Reply:
Oh ok I see. So this is the or one of the reason why we try to always put types in a specific type file instead of keeping them close to where they originate from?

Reply:
Exactly.